.clang-tidy

```diff
@@ -28,7 +28,6 @@ cppcoreguidelines-*,
 -cppcoreguidelines-non-private-member-variables-in-classes,
 -facebook-hte-RelativeInclude,
 hicpp-exception-baseclass,
-hicpp-avoid-goto,
```

Comment:
We shouldn't be against `goto` just because its name is `goto`. We should only be against the usage of `goto` that makes code less readable. C++ is not a perfect language, and using `if`, `switch`, `while`, etc. does not always produce the most readable code.